package.json

```
@@ -1,11 +1,12 @@
 {
   "name": "@axelar-network/axelar-cgp-solidity",
-  "version": "6.0.0",
+  "version": "6.0.1",
```

```suggestion
  "version": "6.0.0",
```
haven't released v6 yet